fatal error: sys/cbprintf_enums.h: No such file or directory

**Describe the bug**
The following commit is causing a compiler error when downstreaming
https://github.com/zephyrproject-rtos/zephyr/commit/aff6e8b2f773cfb848b4ca1394fadd7a1280e4c2

Please also mention any information which could help others to understand
the problem you're facing:
 - include looks to be missing `zephyr` prefix.  
 - I think it should look like `#include <zephyr/sys/cbprintf_enums.h>`

**To Reproduce**
Steps to reproduce the behavior:
1. Compile code

**Expected behavior**
Successful compile

**Impact**
What impact does this issue have on your progress (e.g., annoyance, showstopper): Annoyance

**Logs and console output**
```
[11/356] Building C object CMakeFiles/app.dir/mnt/host/source/src/platform/ec/zephyr/subsys/ap_pwrseq/ap_power_interface.c.obj
                 from /mnt/host/source/src/platform/ec/zephyr/subsys/ap_pwrseq/include/ap_power_host_sleep.h:10,
                 from /mnt/host/source/src/platform/ec/zephyr/subsys/ap_pwrseq/include/x86_non_dsx_common_pwrseq_sm_handler.h:15,
                 from /mnt/host/source/src/platform/ec/zephyr/subsys/ap_pwrseq/ap_power_interface.c:7:
/mnt/host/source/src/third_party/zephyr/main/include/zephyr/sys/cbprintf.h:228:10: fatal error: sys/cbprintf_enums.h: No such file or directory
FAILED: CMakeFiles/app.dir/mnt/host/source/src/platform/ec/zephyr/subsys/ap_pwrseq/ap_power_interface.c.obj 
  228 | #include <sys/cbprintf_enums.h>
```

**Environment (please complete the following information):**
 - OS: Linux
 - Toolchain: Zephyr SDK
 - Commit SHA or Version used: aff6e8b2f773cfb848b4ca1394fadd7a1280e4c2

--- a/include/zephyr/sys/cbprintf.h
+++ b/include/zephyr/sys/cbprintf.h
@@ -225,7 +225,7 @@ BUILD_ASSERT(Z_IS_POW2(CBPRINTF_PACKAGE_ALIGNMENT));
 
 /**@} */
 
-#include <sys/cbprintf_enums.h>
+#include <zephyr/sys/cbprintf_enums.h>
 
 /** @brief Signature for a cbprintf callback function.
  *

